apps/sseramemes/src/tweetMeme.ts

```diff
@@ -61,10 +61,9 @@ export const tweetMeme = async (message: Message | PartialMessage) => {
 
   const mediaIds = mediaId ? [mediaId] : undefined;
   
-  message=message.replace(/@\\/mg,'');
-  message=message.replace(/(@[\w]{1,})#[\d]{1,}/mg,'$1');
+  const messageContent = message.content.replace(/@\\/mg,'').replace(/(@[\w]{1,})#[\d]{1,}/mg,'$1');
```

Comment:
Twitter usernames must be more than 4 characters long and can be up to 15 characters or less.

https://help.twitter.com/en/managing-your-account/change-twitter-handle